pkg/qe-tests/cypress/integration/tests/Rhv/editProvider.ts

```diff
@@ -0,0 +1,21 @@
+import { editProviderData, testData } from './config_separate_mapping_rhv';
+import { login } from '../../../utils/utils';
+import { providerRhv } from '../../models/providerRhv';
+import { RhvProviderData } from '../../types/types';
+
+describe('Creating provider and deleting', () => {
```

Comment:
CRUD (Create, Read, Update and Delete) operations is a better description for this test suite .

describe('Provider CRUD Operations', () => {

Reply:
Modified the descibe :)